changelog-maker.js

```diff
@@ -51,6 +58,16 @@ function replace (s, m) {
   return s
 }
 
+function showUsage () {
+  var usage = fs.readFileSync('README.md', 'utf8')
+    .replace(/[\s\S]+(## Usage\n[\s\S]*)\n## [\s\S]+/m, '$1')
+  if (process.stdout.isTTY)
```

Comment:
I usually make a habit of putting braces around an `if` if the lines within it are >1, even if it's a single statement, no big deal here but just FYI